src/systems/filecoin_markets/storage_market/_index.md

```diff
@@ -3,17 +3,120 @@ menuTitle: Storage Market
+8. The `StorageProvider` checks the voucher and verifies the CID matches a storage deal proposal it has received and verified but not put on chain already. If so, it accepts the data transfer request from the `StorageClient`.
+9. The `Data Transfer Module` now transfers for the payload data to be stored from the  `StorageClient` to the `StorageProvider` using `GraphSync`.
+10. Once complete, the `Data Transfer Module` notifies the `StorageProvider`
+11. The `StorageProvider` recalculates the piece commitment (`CommP`) from the data that was transferred and verifies it matches the piece commitment in the the `StorageDealProposal`
+
+# Publishing
+
+Data is now transferred, both parties have agreed, and it's time for the `StorageProvider` to publish the deal.
+
+12. First, the `StorageProvider` adds collateral for the deal as needed to the `StorageMarketActor` (using `AddBalance`)
+13. Now, the `StorageProvider` calls `PublishStorageDeals` on the `StorageMarketActor` to publish the deal. It sends the StorageDealProposal signed by the client as well as its own signature in the message to publish the storage deal.
+14. For convenience, the `StorageProvider` responds sends a message to the `StorageClient` on the `Storage Deal Protocol` with the CID of the message it is
+putting on chain
+15. If all goes well, the `StorageMarketActor` responds with an on chain `DealID` for the published deal.
+
+
+Finally, the client verifies the deal.
+
+16. The `StorageClient` queries the node for the CID of the message published on chain (sent by the provider). It then inspects message parameters to make sure they match the expected deal
+
+# Handoff
+
+Now that a deal is published, it needs to be stored, sealed, and proven in order for the provider to be paid. (see {{<sref storage_deal>}} for more information about how deal payments are made) These later stages of a deal are handled by the {{<sref storage_mining_subsystem>}}. So the final task for the Storage Market is to handoff to the Storage Mining Subsystem
+
+17. The `StorageProvider` writes the serialized, padded piece to a shared {{<sref filestore>}}. 
+18. The `StorageProvider` calls `HandleStorageDeal` on the `StorageMiner` with the published `StorageDeal` and filestore path.
+
+A note order of operations: the only requirement to publish a storage deal with the `StorageMarketActor` is that the `StorageDealProposal` is signed by the `StorageClient`, the publish message is signed by the `StorageProvider`, and both parties have deposited adequate funds/collateral in the `StorageMarketActor`. As such, it's not required that the steps listed above happen in exactly the order listed above. However, the above order is *recommended* because it generally minimizes the ability of either party to act maliciously.
```

Comment:
We should also mention that it is recommended for StorageProvider to send back the signed Deal message to the client before they call PublishStorageDeals so that client knows that its deal has been signed and the client can choose to publish the deal themselves. 

Note that we might want to relax the constraint that it is okay to publish a StorageDeal that already exists  and not abort as long as it's the same deal cc @jzimmerman .